### PR TITLE
Create enginecreations.json

### PR DESCRIPTION
**Pull request by akk1to**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/enginecreations.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Engine Creations(TM) main pages",
+  "repo": "https://github.com/akk1to/enginecreations.github.io",
+  "owner": {
+    "username": "Engine Creations(TM)",
+    "email": "contact@enginecreations.is-a.dev",
```
Comment (Stef-00012) [inline domains/enginecreations.json:6]: This file has no MX record so it can not receive mails

File: domains/enginecreations.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Engine Creations(TM) main pages",
+  "repo": "https://github.com/akk1to/enginecreations.github.io",
+  "owner": {
+    "username": "Engine Creations(TM)",
```
Comment (Stef-00012) [inline domains/enginecreations.json:5]: Username should be your github username

**Review by Stef-DP — DISMISSED** [review]

File: domains/enginecreations.json (hunk 1 above)
Comment (akk1to) [reply domains/enginecreations.json]: resolved the change

File: domains/enginecreations.json (hunk 2 above)
Comment (akk1to) [reply domains/enginecreations.json]: resolved the change

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**